relentless/simulate/lammps.py

```diff
@@ -437,6 +437,11 @@ def to_commands(self, sim):
 class MinimizeEnergy(LAMMPSOperation):
     """Runs an energy minimization until converged.
 
+    Valid **options** include:
+
+    - **max_evaluations** (`int`) - the maximum number of time steps the minimizer
+      is allowed to run per iteration. Defaults to 100.
```

Comment:
This documentation is not quite right for LAMMPS. In LAMMPS, `max_evaluations` is:

> maxeval = max number of force/energy evaluations

I would recommend that the default is not fixed at 100. Instead, maybe set it to 100 / iteration (i.e., `max_evaluations = 100 * max_iterations`).

Reply:
(You would need to modify the code for setting the default below too.)